modules/bringauto_docker/DockerTools.go

```diff
@@ -0,0 +1,30 @@
+package bringauto_docker
+
+import (
+	"bringauto/modules/bringauto_process"
+	"bytes"
+)
+
+func IsDefaultPortAvailable() bool {
+	var outBuff, errBuff bytes.Buffer
+
+	process := bringauto_process.Process{
+		CommandAbsolutePath: DockerExecutablePathConst,
+		Args: bringauto_process.ProcessArgs{
+			ExtraArgs: &[]string{
+				"container",
+				"ls",
+				"--filter",
+				"publish=1122",
```

Comment:
Maybe crawl all the code an all of these shared constants gather into one module and use it in places? (so there is exactly one place where constants are defined and documented....)

Reply:
I would create bringauto_const module which will gather all shared constants between module.

Reply:
oukej.

Reply:
Done.